QUARKUS-728.md

```diff
@@ -0,0 +1,70 @@
+# Quarkus-728 - Add support for OpenJDK 17
+
+JIRA link: https://issues.redhat.com/browse/QUARKUS-728
+
+The goal is to verify OpenJDK 17 integration
+
+## Scope of the testing
+Test development will focus on  
+ - New specifict scenarios will be created to cover the integration of new objects as “sealed” or “records” classes with several extensions as Hibernate or RestEasy.
+ - Current scenarios must work with Java 17
+ - S2i (source to image) must works with Java 17  
+
+### Impact on testsuites and testing automation:
+ - Ensure this coverage works in JVM mode
+ - NATIVE mode is not required in Java 17 
+ - Ensure this coverage works in Openshift (s2i)
+ - New test suite will be created in order to develop specifict Java 17 scenarios
+ - New test suite will use Gradle as a build tool
+ - The following Jenkins jobs must be duplicated to support Java 17:
+   - RHBQ
```

Comment:
move the ones with `?` into separate section and mention that we will need to decide in the wider QE team discussion if these jobs need Java 17 equivalent or not

Reply:
I have re-written all of this section, but I add a note, where I explain that the final Jenkins job list must be discussed by the QE team.